### DISCUSSION TITLE
ERROR when i start a container

### DISCUSSION THREAD

**Discussion by miguecrty** [containers/podman#22419] [General]
I've installed the latest version of Podman from source code on my Ubuntu 22.04 system, but when I try to start a container, I get this error.
I'm going to put the Podman installation script that I've used (It's a .txt file because GitHub doesn't let me use the .sh extension):
![bug](https://github.com/containers/podman/assets/125964187/3d32e109-a843-4b24-b95a-4831e74adee1)

[instalacion.txt](https://github.com/containers/podman/files/15022970/instalacion.txt)
87/6c3c60a0-d9dc-49a3-b70d-49452a60b85e)

**Comment by Luap99** [comment]
you have to install netavark or as the error says if it is already installed update containers.conf to point to the location of netavark.

**Reply by miguecrty** [reply]
I have installed container-networking-plugins because netavark its not available in ubuntu 22.04 with arvh amd64.

**Reply by Luap99** [reply]
podman 5.0 dropped support for cni by default, it can be enabled via `cni` build tag but we upstream no longer support it.

**Reply by miguecrty** [reply]
so what can i do to fix that error?
because if i install podman using apt install its install podman v3.4.4 and i need podman +v.4
